roles/pulp3-devel/tasks/main.yml

```diff
@@ -0,0 +1,108 @@
+---
+- block:
+
+  - name: Install several useful packages
+    package:
+      name:
+        - bash-completion
+        - dnf-utils
+        - dstat
+        - fd-find
+        - fedora-easy-karma
+        - fpaste
+        - fzf
+        - git
+        - htop
+        - httpie
+        - iotop
+        - jnettop
+        - jq
+        - ncdu
+        - python-django-bash-completion
+        - python-ipdb
+        - python-rpdb
+        - python3-setuptools
+        - python3-virtualenvwrapper
+        - redhat-lsb-core
+        - ripgrep
+        - screen
+        - telnet
```

Comment:
We used to have an alias that used this, but I don't see it "pdebug". This is only necessary if we have rpdb installed. I say,we remove this.